src/test/java/co/rsk/federate/FedNodeRunnerTest.java

```diff
@@ -443,69 +439,72 @@ void test_with_no_config() throws Exception {
 
     @Test
     void run_whenHsmVersionIsLowerThanThreeAndDifficultyTargetConfigIsNotPresent_shouldThrowException() throws Exception {
-        SignerConfig btcSignerConfig = getHSMBTCSignerConfig(2);
-        when(btcSignerConfig.getConfig().getString(DIFFICULTY_TARGET.getPath()))
-           .thenThrow(new ConfigException.Missing(DIFFICULTY_TARGET.getPath()));
+        int version = 2;
+        when(hsmBookkeepingClient.getVersion()).thenReturn(version);
+        when(hsmBookkeepingClient.getBlockchainParameters()).thenThrow(
+            new HSMUnsupportedTypeException("PowHSM version: " + version));
+        Config config = TestConfigBuilder.builder()
+            .withValue("type", "hsm")
+            .withValue("host", "127.0.0.1")
+            .withValue("port", 9999)
+            .withValue("keyId", "m/44'/0'/0'/0/0")
+            .withValue("socketTimeout", 20000)
+            .withValue("maxAttempts", 3)
+            .withValue("intervalBetweenAttempts", 2000)
+            .withValue("bookkeeping.informerInterval", 500000L)
+            .withValue("bookkeeping.maxAmountBlockHeaders", 1000)
+            .withValue("bookkeeping.maxChunkSizeToHsm", 100)
+            .withValue("bookkeeping.stopBookkeepingScheduler", true)
```

Comment:
Perhaps all these could also be grouped somehow

Reply:
I feel like if we add them all in the `withHsmSigner` method then it could be too granular and not very flexible. wdyt?

Reply:
That's true.

Perhaps a method `withBookkeepingInfo` and receive all the data as params? To group things a bit and not force the one who is using this to know the name of each field that needs to be assigned